CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Delta Chat iOS Changelog
 
+## 1.17.0
+2021-03
+
+* new mailinglist and better bot support
+* support reliable notifications at least every 20 minutes
+* add option to view original-/html-mails
+* allow dialing on tapping a phone number
+* support more audio formats
```

Comment:
is this done by the core? 
What we did UI-wise was only to show errors if a audio format is not supported.

Reply:
ah, i mixed that up, thanks for the hint.